samples/app/src/main/res/drawable/appyx.xml

```diff
@@ -0,0 +1,37 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
@KovalevAndrey does this work on lollipop? Or should we change our min SDK for the samples app to a version that supports this without crashing?